### ISSUE TITLE
Sort title is not working.

### ISSUE THREAD

**Issue by GotekSC** [jellyfin/jellyfin#1321]
**Describe the bug**
When sorting by name, the sort title gets ignored.

**To Reproduce**
1. Go to Movies
2. Click on 'The Empire Strikes Back'
3. Rename Sort title to: Star Wars: Episode V - The Empire Strikes Back 
4. Empire Strikes Back still gets sorted by E.

**Expected behavior**
After renaming the sort title, Empire Strikes Back should get sorted between Star Wars: Episode IV - A New Hope, and Star Wars: Episode VI - Return of the Jedi.

**Screenshots**
[Edit metadata screen](https://i.imgur.com/wxltjbN.png)

**System (please complete the following information):**
 - OS: Docker on ubuntu 18.0.4
 - Browser: Firefox
 - Jellyfin Version: 10.3.1
 - Reverse proxy: none

**Additional context**
I like to title them Star Wars, The Empire Strikes back and Return of the Jedi. So this is a bummer.

**Comment by bugfixin** [comment]
I'm not able to reproduce this - could you screenshot the edit metadata screen so I can try reproducing with more fields?

**Comment by GotekSC** [comment]
Thanks for the response. Here you go:
https://i.imgur.com/wxltjbN.png

**Comment by GotekSC** [comment]
Alright. The problem resolved itself for me. 

While I was preparing the screenshot, I noticed that the tmdb box sets plugin recreated the star wars collection. I used to remove the movies out of the collection manually.

So being annoyed at that, I removed the plugin, and deleted the residual collections. And now sorting by title works.

So I presume that this issue was isolated to items "belonging" to collections by tmdb box sets, but being manually removed.
